### DISCUSSION TITLE
What is Copula ? (About DeepVAR and GPVAR)

### DISCUSSION THREAD

**Discussion by ken-take-it-so-so** [awslabs/gluonts#2303] [Q&A] [answered]
Hello. I'm still learning Python and GluonTS.  
I want to use "gluonts.model.deepvar.DeepVAREstimator" and "gluonts.model.gpvar.GPVAREstimator". According to GluonTS documentation, these models are introduced in this paper: [High-Dimensional Multivariate Forecasting with Low-Rank Gaussian Copula Processes](https://arxiv.org/abs/1910.03002)  
However, I don't know what Copula is. A below figure is in the paper.

![vec-lstm](https://user-images.githubusercontent.com/109563059/190342248-30297005-ec16-402b-b723-913a12587b42.png)

When I want to try "Vec-LSTM-lowrank-Copula", how should I set arguments in DeepVAREstimator ? Especially, could you teach me how to use "Copula" ?  
Moreover, in [gluonts.model.gpvar package](https://ts.gluon.ai/stable/api/gluonts/gluonts.model.deepvar.html), it seems that GPVAR implements "GP-Copula". Then, when I want to try "GP" and "GP-scaling", how should I set arguments in GPVAR ?  

I would appreciate your cooperation.

**Comment by jgasthaus** [comment, marked as answer]
"Copula" here refers to whether the marginal distributions are transformed to be (approximately) Gaussian before the model is applied. This is controlled by the `use_marginal_transformation` parameter which can be set when constructing both `GPVar` and `DeepVar` estimators. Setting this parameter to `True` will give you the Copula variants. Scaling is enabled by setting `scaling=True`.

https://github.com/awslabs/gluonts/blob/015ff06ae78ef78d7f5d73ccceef7ceb9d4db095/src/gluonts/mx/model/gpvar/_estimator.py#L132

**Reply by ken-take-it-so-so** [reply]
@jgasthaus, thank you for your response. Lastly, let me ask an additional question. Should I set `scaling=False` when I try "-Copula" model in the figure ?

**Reply by jgasthaus** [reply]
Yes. Additional scaling is not necessary when using the marginal transform.

**Reply by ken-take-it-so-so** [reply]
Thank you ! I enjoyed chatting with you !!
